devel/cdecl/Portfile

```diff
@@ -14,19 +15,31 @@ description         Composing and deciphering C (or C++) type declarations \
                     or casts.
 long_description    cdecl (see-deh-kull) is a program for composing \
                     and deciphering C (or C++) type declarations or casts, \
-                    a.k.a. \"gibberish.\" It can be used interactively on a \
-                    terminal or accept input from either the command line \
+                    a.k.a. “gibberish”. It can be used interactively \
+                    on a terminal or accept input from either the command line \
                     or standard input.
 
-checksums           rmd160  f3b89d81fc801158fa4eca7876b6bf40789232ff \
-                    sha256  53f60863e8cb597234ff08f619b47f7d7e11fc8d83543821b0e88acf11a714e4 \
-                    size    299046
+checksums           rmd160  5536a94ead950676bd2ed2c7769ea92c8f22a8af \
+                    sha256  ba17cde63b42e772bcf7b8a67bc8979eb9df49b7c0299ea35d885eac7d4a1cd2 \
+                    size    779556
 
 depends_build-append    port:autoconf \
                         port:automake
 
 depends_lib-append      port:ncurses \
-                        port:readline
+                        port:readline \
+                        port:flex \
+                        port:bison
+
+# gcc-4.2 fails to build it
+compiler.blacklist-append *gcc-4*
+
+if {${os.major} == 10 && ${build_arch} eq "ppc"} {
```

Comment:
is this really only needed on 10.6 or are earlier OSes also affected?

Reply:
> is this really only needed on 10.6 or are earlier OSes also affected?

@reneeotten This is only needed for 10.6.8 Rosetta (see the comment line). Neither 10.5.8, nor 10.6 PPC need this.

Reply:
@reneeotten This is only needed for 10.6.8 Rosetta (see the comment line). Neither 10.5.8, nor 10.6 PPC need this.